README.md

```diff
@@ -21,6 +21,7 @@ Place the content of this repository in **owncloud/apps/oauth2**.
     2. `client_id` (required): the client identifier obtained when registering the client.
     3. `redirect_uri` (required): the redirection URI specified when registering the client.
     4. `state` (optional): can be set by the client "to maintain state between the request and callback" ([RFC 6749](https://tools.ietf.org/html/rfc6749#section-4.1.1)).
+    5. `user` (optional): can be set to indicate the username of the resource owner 
```

Comment:
Maybe to add for proper reference: 
``([RFC 6749](https://tools.ietf.org/html/rfc6749#section-4.3.2)).``

Reply:
@mmattel that part of the RFC is for the step no 4. in this documentation. the reference for this section is https://tools.ietf.org/html/rfc6749#section-4.1.1 and its linked in the heading. That sectionRFC does not mention the user name